TL improvements (#314)
07th-mod/higurashi-console-arcs: Update/omot_010_togoodend.txt

//rその際、入江が『東京』に向けて作り上げたレポートの一部を秘かに盗み見る機会があったのだが、そこに書かれてあった『オヤシロさま』という文言――。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その際、入江が『東京』に向けて作り上げたレポートの一部を秘かに盗み見る機会があったのだが、そこに書かれてあった『オヤシロさま』という文言——。",

> 			NULL, "At the time, I accidentally snuck a read part of a report Irie made for \"Tokyo\" and found the name \"Oyashiro-sama\" written there...", Line_Normal);

Reviewer: ```suggestion
			NULL, "At the time, I discretely snuck a glance at part of a report Irie made for \"Tokyo\" and found the name \"Oyashiro-sama\" written there...", Line_Normal);
```

	ClearMessage();

//rレナは、自分にまつわる異常状態を全て、幼い時に聞いた『オヤシロさまの崇り』によるものだと解釈して、その原因を自らの家族愛の至らなさに求めたのだという。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "レナは、自分にまつわる異常状態を全て、幼い時に聞いた『オヤシロさまの崇り』によるものだと解釈して、その原因を自らの家族愛の至らなさに求めたのだという。",

> 			NULL, "It said that Rena interpreted all her abnormal circumstances to a concept she heard when she was young called \"the curse of Oyashiro-sama\" resulted from her lack of familial love.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It said that Rena interpreted all her abnormal circumstances through a concept she heard when she was young called \"the curse of Oyashiro-sama\", resulting from her lack of familial love.", Line_Normal);
```

	ClearMessage();

//r…これは、安易に責任転嫁する年相応の子供では決して起こりえないことで、それだけレナが心優しく、家族を愛し…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…これは、安易に責任転嫁する年相応の子供では決して起こりえないことで、それだけレナが心優しく、家族を愛し…。",

> 			NULL, "...It's normal for a child of her age to settle on something or someone to push responsibility on to, but Rena didn't do such a thing because paradoxically she really did love her family...", Line_Normal);

Reviewer: ```suggestion
			NULL, "...It's normal for a child of her age to settle on something or someone to push responsibility on to, but Rena didn't because she really did love her family...", Line_Normal);
```

	ClearMessage();

//r…そして、自立心の高さゆえに内罰的な思考へと変化していったためだと、レポートにはまとめられていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そして、自立心の高さゆえに内罰的な思考へと変化していったためだと、レポートにはまとめられていた。",

> 			NULL, "...and because her high degree of independence caused her to focus her thoughts on blaming herself. The report summarized so.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...And because her high degree of independence caused her to focus her thoughts on blaming herself. That's how the report summarized it.", Line_Normal);
```

//rもし、今回レナが主張するオヤシロさまが、文字通りレナの潜在的な思いであったとしたら、レナに敵意を収めろと言うだろうか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "もし、今回レナが主張するオヤシロさまが、文字通りレナの潜在的な思いであったとしたら、レナに敵意を収めろと言うだろうか。",

> 			NULL, "If the Oyashiro-sama that Rena is so insistent on is actually a manifestation of her subconscious, why would it stop her being hostile?", Line_Normal);

Reviewer: ```suggestion
			NULL, "If the Oyashiro-sama that Rena is so insistent on is actually a manifestation of her subconscious, why would it stop her from becoming hostile?", Line_Normal);
```

//rとぼとぼと歩く足音はとても物悲しく…頼りない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "とぼとぼと歩く足音はとても物悲しく…頼りない。",

> 			NULL, "The sound of our footsteps was too pathetic... and too pale.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The sound of our footsteps was too pathetic... and too uneasy.", Line_Normal);
```

			NULL, "\"...Hey, Rena.", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100942", 256, TRUE);
	OutputLine(NULL, "オヤシロさまは、間違ってなかったと思う。",

> 			NULL, " I think Oyashiro-sama told you correct things.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I think what Oyashiro-sama told you wasn't wrong.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100943", 256, TRUE);
	OutputLine(NULL, "ただ、それをレナが勘違いしていただけなんじゃないのか？」",

> 			NULL, " But maybe you just misunderstood them?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " But maybe you just misunderstood it?\"", Line_Normal);
```

		ModPlayVoiceLS(4, 5, "ps3/s14/05/vt4ba_rika1130", 256, TRUE);
	}
	OutputLine(NULL, "「頼りたいと思うことを、誰も迷惑には感じたりしない。それに気づいて欲しかった…でなければ、きっと後悔するから…」",

> 			NULL, "\"Nobody wants to feel like a nuisance for depending on others. You should find that...because otherwise you'd end up feeling regret over it...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Nobody wants to feel like a nuisance for depending on others. You should realize that... because otherwise you'd end up feeling regret over it...\"", Line_Normal);
```

//r……今まで私は自分の不幸を、『オヤシロさま』の祟りだと思っていた。だから、私は『オヤシロさま』の言うことをこれからは信じよう、守っていこうと誓ったんだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "……今まで私は自分の不幸を、『オヤシロさま』の祟りだと思っていた。だから、私は『オヤシロさま』の言うことをこれからは信じよう、守っていこうと誓ったんだ。",

> 			NULL, "...Up until now, I always considered my misfortune the curse of \"Oyashiro-sama.\" And so, I vowed to believe and keep what \"Oyashiro-sama\" told me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Up until now, I always considered my misfortune the curse of \"Oyashiro-sama.\" And so, I vowed to believe and obey what \"Oyashiro-sama\" told me.", Line_Normal);
```

//r二人はそう言って、顔を見合わせ無言で頷く。ご馳走、魅音の家で宿泊…？　つまり、最初からここまで、準備していたということか…？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "二人はそう言って、顔を見合わせ無言で頷く。ご馳走、魅音の家で宿泊…？　つまり、最初からここまで、準備していたということか…？",
			NULL, "They exchanged glances and silently nodded to each other as they said that. A feast and overnight stay at Mion's house...? So, they had this planned out from the start...?", Line_Normal);
	ClearMessage();

//rその、あまりの手際のよさに私も思わず呆気に取られた。…圭一。どこまであなたは、私を感動させれば気がすむというの…？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その、あまりの手際のよさに私も思わず呆気に取られた。…圭一。どこまであなたは、私を感動させれば気がすむというの…？",

> 			NULL, "I was completely taken aback by his amazing forethought. ...Keiichi. Just how far are you willing to go to make me impressed...?", Line_Normal);

Reviewer: ```suggestion
			NULL, "I was completely taken aback by his amazing forethought. ...Keiichi. Just how far are you willing to go to impress me...?", Line_Normal);
```
